platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/datasource/VirtualLeafBytes.java

```diff
@@ -44,6 +44,8 @@
  */
 public class VirtualLeafBytes<V> {
 
+    public static final FieldDefinition FIELD_ONE_OF_STATE_ITEM =
```

Comment:
Usually field definitions like this are named `FIELD_messageType_fieldName`, so this one should be `FIELD_MERKLELEAF_STATEITEM`